### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by jmrodri**
while working on vendoring doc, I noticed a typo I copied from
bundle-lib. Ran through all of the comments to check their spelling.

### CODE REVIEW

**Review by eriknelson — APPROVED** [review]
Are you using a spellchecker tool for these?

**Review by shawn-hurley — APPROVED** [review]

File: crd/conversions.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ func ConvertServiceBindingToCRD(bi *bundle.BindInstance) (v1.ServiceBindingSpec,
 // along with its id (which is often the ServiceBinding's name), and will convert
 // these into a bundle BindInstance.
 func ConvertServiceBindingToAPB(bi v1.ServiceBindingSpec, id string) (*bundle.BindInstance, error) {
-	// TOOD: Same as above, accept the full ServiceBinding?
```
Comment (djzager) [inline crd/conversions.go]: Heh.

**Review by djzager — APPROVED** [review]
I appreciate getting to see these. Makes my day.

**Comment by jmrodri** [comment]
@eriknelson yes, I use `ispell`. Here is my method:
```
git grep "\/\/ " > comments
ispell comments
```
In another window I open up the files listed in the file that match the errored line.  Same approach for logs:

git grep "log\." > logs

**Pull request merged**